biz.aQute.bndlib/src/aQute/bnd/classfile/ConstantPool.java

```diff
@@ -888,4 +986,14 @@ public void write(DataOutput out) throws IOException {
 			}
 		}
 	}
+
+	@Override
+	public Iterator<Object> iterator() {
+		return Arrays.asList(pool)
+			.iterator();
```

Comment:
`return stream().iterator();` which uses the (updated) stream method that avoids returning the invisible 0-index entry.